app/views/layouts/darkswarm.html.haml

```diff
@@ -14,8 +14,7 @@
       = favicon_link_tag "/favicon-staging.ico"
     %link{href: "https://fonts.googleapis.com/css?family=Roboto:400,300italic,400italic,300,700,700italic|Oswald:300,400,700", rel: "stylesheet", type: "text/css"}
     %link{href: asset_pack_path("media/fonts/OFN-v2.woff"), rel: "preload", as: "font", crossorigin: "anonymous"}
-    = render "layouts/matomo_tag"
-    = render "layouts/matomo_tracking"
```

Comment:
Can we remove the tracking partial now? I can't find any usage.

Reply:
You mean the actual files?
```
app/views/layouts/_matomo_tag.html.haml
app/views/layouts/_matomo_tracking.html.haml
```
I think it could be useful to be able to add these selectively... It's not entirely clear if there is any disadvantage of using one tag for everything. I'll need to double check with Matomo customer support.

Reply:
I'm moving this to in Dev till I'm sure.